packages/terra-demographics-banner/src/DemographicsBanner.jsx

```diff
@@ -32,6 +33,10 @@ const propTypes = {
    */
   // eslint-disable-next-line react/forbid-prop-types
   identifiers: PropTypes.object,
+  /**
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
```

Comment:
We should add the `@private` syntax so this doesn't show up in the generated prop-types documentation.